.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: release
+on:
+  push:
+    tags:
+      - "*"
+permissions:
+  contents: write
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: build
+        run: mkdir .release-artifacts && cp $(bazel build --show_result=9999 target-determinator:all driver:all 2>&1 | grep 'bazel-out') .release-artifacts/
+      - name: release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: .release-artifacts/*
```

Comment:
add `generate_release_notes: true` since it's amazingly effective and you won't have to think about changelog